core/Tracker/PageUrl.php

```diff
@@ -174,7 +174,7 @@ protected static function cleanupHostAndHashTag($parsedUrl, $idSite = false)
         }
 
         if (!empty($parsedUrl['host'])) {
-            $parsedUrl['host'] = Common::mb_strtolower($parsedUrl['host'], 'UTF-8');
+            $parsedUrl['host'] = Common::mb_strtolower($parsedUrl['host']);
```

Comment:
This change is un-related and not for this PR?

Reply:
Yes. Unrelated. It removes an useless parameter as `Common::mb_strtolower` only takes one parameter